scss/mixins/_decoration.scss

```diff
@@ -2,12 +2,12 @@
 @mixin fill( $color: null, $bg: null, $border: null, $gradient: null ) {
     border-color: $border;
     color: $color;
-    background-color: $bg;
+    background: $bg;
```

Comment:
The short hands, like `background`, reset lots of stuff. That line, by itself means:

```scss
selector {
    background-color: $bg; // that's the only things that's set
    background-position: initial; // or 0 0 in older browsers
    background-repeat: initial; // or repeat in older browsers
    background-origin: initial; // or padding-box in older browsers
    background-image: initial; // or none in older browsers
    background-size: initial; // or auto in older browsers
}
```

That's why we specifically use the fully qualified properties.